### ISSUE TITLE
Uncaught Error: Reducer "x" returned undefined during initialization

### ISSUE THREAD

**Issue by gaulucky92** [omnidan/redux-ignore#6]
Hi, 
I'm developing a web app that has a complex data structure, exploding an object and pass its data to many stores (reducers). To reduce the stress of listening to stores and callling mapStateToProps too many times, I planned to use redux-ignore, to help reducers ignoring unnecessary actions (as `post` doesn't need to know about an USER_FETCH action).
Unfortunately, I'm still in the mess as I could not use redux-ignore just like the manual says. It only works when I pass an array of pre-defined actions to ignore, and causes error if I want to use an actionHandler function, as following:
```javascript
combineReducers({
    block: filterActions(block, (action) => action.type.match(/^BLOCK(.*?)$/))
});
```
I want to let all actions whose types begin with "BLOCK" (BLOCK_CREATE, BLOCK_FETCH,...) go through the Block reducer, and ignore the rest, but Redux would warn me:
> Uncaught Error: Reducer "block" returned undefined during initialization. If the state passed to the reducer is undefined, you must explicitly return the initial state. The initial state may not be undefined.

Did I miss something here?

**Comment by artemkaint** [comment]
+1

**Comment by davidkpiano** [comment]
What does your `block` reducer look like?

**Comment by omnidan** [comment]
this should be fixed in [1.2.4](https://github.com/omnidan/redux-ignore/releases/tag/v1.2.4) (thanks @maxmechanic!) - please let me know if there are any more issues :grin:
